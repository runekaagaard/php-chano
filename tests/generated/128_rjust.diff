001- test
001+ Fatal error: Call to undefined method DtlIter::rjust() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/128_rjust.php on line 4