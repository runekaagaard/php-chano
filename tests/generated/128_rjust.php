<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'test')));
foreach ($items as $i) echo $i->input->rjust(10);
