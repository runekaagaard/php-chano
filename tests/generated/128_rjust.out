Fatal error: Call to undefined method DtlIter::rjust() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/128_rjust.php on line 4